builtin/providers/aws/resource_aws_api_gateway_client_certificate.go

```diff
@@ -0,0 +1,129 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayClientCertificate() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayClientCertificateCreate,
+		Read:   resourceAwsApiGatewayClientCertificateRead,
+		Update: resourceAwsApiGatewayClientCertificateUpdate,
+		Delete: resourceAwsApiGatewayClientCertificateDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
Got it! It's usually in a different file - maybe we need to remove that from elsewhere as it is just extra files in the repo :)